src/test/java/stormpot/simulations/TimeSpreadExpirationSim.java

```diff
@@ -25,10 +25,12 @@
  */
 @Sim.Simulation(pools = {BlazePool.class}, measurementTime = 60, output = Sim.Output.summary)
 public class TimeSpreadExpirationSim extends Sim {
+  @SuppressWarnings("rawtypes") // Not possible to have a generic array. It might be possible to make @Conf work with a
```

Comment:
I'm never entirely sure that I understand generics correctly, so please double check this PR before merge. In particular, I'm not completely sure of the comment I added here (and the `@SuppressWarnings`).